Formula/s/stylua.rb

```diff
@@ -1,8 +1,8 @@
 class Stylua < Formula
   desc "Opinionated Lua code formatter"
   homepage "https://github.com/JohnnyMorganz/StyLua"
-  url "https://github.com/JohnnyMorganz/StyLua/archive/refs/tags/v0.20.0.tar.gz"
-  sha256 "f4a27b12669953d2edf55b89cc80381f97a2dfa735f53f95c6ae6015c8c35ffb"
+  url "https://github.com/JohnnyMorganz/StyLua/archive/refs/tags/v2.0.0.tar.gz"
```

Comment:
this is straight to 2.0.0

https://redirect.github.com/JohnnyMorganz/StyLua/issues/459

Reply:
Because 1.0 is for chumps :)

Reply:
For the record, the version number _did_ jump from v0.20.0 to v2.0.0, with no other versions in between.

Reply:
indeed, but I feel like it should be 1.0.0 still though.

Reply:
Yep, but that's the dev's decision, not ours.